src/quill-cursors/quill-cursors.ts

```diff
@@ -110,6 +116,26 @@ export default class QuillCursors {
   private _registerDomListeners(): void {
     const editor = this.quill.container.getElementsByClassName('ql-editor')[0];
     editor.addEventListener('scroll', () => this.update());
+
+    editor.addEventListener('mousemove', throttle(this._toggleNearCursors.bind(this), 100));
```

Comment:
This feels pretty heavy-handed to me. Using a throttle like this is a bit of a code smell.

Did you try `mouseenter` and `mouseleave`? I think from experience they can be a bit flaky, so no big deal if they didn't work.

However, I'd rather we then move to use `mouseover`, and specifically attach to the elements we care about our mouse being inside. That would at least target our events a bit better, and would avoid needing to have the concept of being "near" the cursor.